src/vcSettingsUI.cpp

```diff
@@ -965,6 +965,8 @@ void vcSettingsUI_BasicMapSettings(vcState *pProgramState)
       pProgramState->activeProject.pFolder->ChangeProjection(zone);
     }
   }
+
+  ImGui::Checkbox(vcString::Get("settingsMapsMapTiles"), &pProgramState->settings.maptiles.mapEnabled);
```

Comment:
This function is also called by the settings UI, is there now two checkboxes to enable/disable map tiles?

Reply:
yes. someone just wanted a checkbox in the shortcut window for convenience

Reply:
I don't think it makes sense to have two checkboxes for the same thing in the settings UI though. Do you?

Reply:
No so onew of these boxes is in the settings window, and the other checkbox is in the little Map shortcuts window in the bottom left

Reply:
Are you sure? This function is called by both of those systems.

Reply:
yes there's another little window called "map settings" but it isn't actually part of the settings UI. It's in the bottom right of the screen. This is where I've added the checkbox

Reply:
![image](https://user-images.githubusercontent.com/11887457/86075595-b372e400-bacb-11ea-9642-f891e1786c6b.png)
This is what I'm talking about.

Reply:
oh what i didnt see that when i tested it

Reply:
OK i get what ur saying now. I thought they were 2 separate functions